auto_round/script/mllm.py

```diff
@@ -47,7 +47,7 @@ def __init__(self, *args, **kwargs):
         self.add_argument("--asym", action='store_true',
                             help=" asym quantization")
 
-        self.add_argument("--dataset", required=True, type=str,
+        self.add_argument("--dataset", type=str, default=None,
```

Comment:
why there is no default dataset?

Reply:
Cause for llava dataset, the argument 'dataset' must be a path

Reply:
how about using one in https://huggingface.co/datasets?search=llava